engine/inputdata/blueprint.ftl

```diff
@@ -21,6 +21,238 @@
+                    "Children" : referenceConfig.Attributes
+                }
+            ]]
+        [/#if]
+    [/#list]
+
+    [#local settingsChildren = [
+        {
+            "Names" : "Settings",
+            "Children" : [
+                {
+                    "Names" : "Accounts",
+                    "Children" : [
+                        {
+                            "Names" : "*",
+                            "Types" : ANY_TYPE
+                        }
+                    ]
+                },
+                {
+                    "Names" : "Products",
+                    "Children" : [
+                        {
+                            "Names" : "*",
+                            "Types" : ANY_TYPE
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "Names" : "Builds",
+            "Children" : [
+                {
+                    "Names" : "Accounts",
+                    "Children" : [
+                        {
+                            "Names" : "*",
+                            "Types" : ANY_TYPE
+                        }
+                    ]
+                },
+                {
+                    "Names" : "Products",
+                    "Children" : [
+                        {
+                            "Names" : "*",
+                            "Types" : ANY_TYPE
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "Names" : "Sensitive",
+            "Children" : [
+                {
+                    "Names" : "Accounts",
+                    "Children" : [
+                        {
+                            "Names" : "*",
+                            "Types" : ANY_TYPE
+                        }
+                    ]
+                },
+                {
+                    "Names" : "Products",
+                    "Children" : [
+                        {
+                            "Names" : "*",
+                            "Types" : ANY_TYPE
+                        }
+                    ]
+                }
+            ]
+        }
+    ]]
+
+    [#local certChildren = [
+        {
+            "Names" : "Certificates",
+            "SubObjects" : true,
+            "Children" : [
+                {
+                    "Names" : "Domain",
+                    "Types" : STRING_TYPE
+                }
+            ]
+        }
+    ]]
+
+    [#local domainChildren = [
+        {
+            "Names" : "Domains",
+            "Children" : [
+                {
+                    "Names" : "*",
+                    "Types" : OBJECT_TYPE,
+                    "Children" : [
+                        {
+                            "Names" : "Stem",
+                            "Types" : STRING_TYPE,
+                            "Mandatory" : true
+                        }
+                    ]
+                },
```

Comment:
I think domain has more properties than just stem. It has a few like Name, Primary etc.  I thought there was some validation for this somewhere

Reply:
Name is added as a part for the Framework Attributes to everything, and I found the other configuration defined in `getDomainObjects()` - I've refactored that now in b463fd8 for your review.

Reply:
Stem is used everywhere? What does it do? that seems really weird

Reply:
Sorry didn't expand where the domain value was being used. All good